### DISCUSSION TITLE
Waha and react native

### DISCUSSION THREAD

**Discussion by DanielTheProgrammer** [devlikeapro/waha#362] [General]
Is it possible to run waha instance in react native application(that will run in mobile)?

**Comment by devlikepro** [comment]
Unlikely, if it can not run docker. 
Under the hood it runs chromium (if we're talking about WEBJS engine), that something Android usually can not do
You can try using NOWEB (no chrome required, it uses websockets), let us know if you succeed!
https://waha.devlike.pro/docs/how-to/engines/

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
